package.xml

```diff
@@ -15,6 +15,12 @@
 
   <buildtool_depend>ament_cmake_core</buildtool_depend>
 
+  <build_depend>rosidl_default_generators</build_depend>
```

Comment:
To match other message packages (e.g. https://github.com/ros2/common_interfaces/blob/master/diagnostic_msgs/package.xml), this should be a `<buildtool_depend>`

Reply:
Done. By the way, I copied this from the ROS 2 tutorial.